commons/SourceMapSupport.ts

```diff
@@ -1,6 +1,6 @@
-import * as SourceMapSupport from 'source-map-support';
+import { install } from 'source-map-support';
 
 if (!Error[Symbol.for('source-map-support')]) {
   Error[Symbol.for('source-map-support')] = true;
-  SourceMapSupport.install({ uncaughtShimInstalled: false });
+  install({ handleUncaughtExceptions: false });
```

Comment:
I find that destructured method/property imports to be much harder to follow in the code base. I realize that this allows for smaller files, but does it matter for a server-side tool like SA?

Reply:
No, it doesn't really matter, but in this specific case, source map support was behaving strangely with the es export * and nothing else is happening in this file, so I thought an exception was ok.

Reply:
Cool. That's a good reason.